src/plugins/intel_cpu/src/nodes/input.cpp

```diff
@@ -92,30 +92,32 @@ struct jit_has_subnormals_base : public jit_generator {
         pop(rsi);
     }
 
-    void check_subnormals(const Xbyak::Reg64& src, const Xbyak::Ymm &mask, const Xbyak::Ymm &zero) {
+    void check_subnormals(const Xbyak::Reg64& src, const Xbyak::Ymm &exponent_mask, const Xbyak::Ymm &mantissa_mask, const Xbyak::Ymm &zero) {
         auto a = ymm1;
         auto b = ymm2;
         auto c = ymm3;
 
         vmovdqu(a, yword[src]);         // load 8 floats
-        vpcmpeqd(b, a, zero);           // if (a == 0) b = 1 else b = 0
-        vpand(c, a, mask);              // c = a & 01111111100000000000000000000000
+        vpand(b, a, mantissa_mask);     // b = a & 00000000011111111111111111111111
+        vpcmpeqd(b, b, zero);           // if (b == 0) b = 1 else b = 0
+        vpand(c, a, exponent_mask);     // c = a & 01111111100000000000000000000000
         vpcmpeqd(c, c, zero);           // if (c == 0) c = 1 else c = 0
         vptest(b, c);                   // if ((!b & c) == 0) CF = 1 else CF = 0
     }
 
-    void check_subnormals(const Xbyak::Reg64& src, const Xbyak::Xmm &mask, const Xbyak::Xmm &zero) {
+    void check_subnormals(const Xbyak::Reg64& src, const Xbyak::Xmm &exponent_mask, const Xbyak::Xmm &mantissa_mask, const Xbyak::Xmm &zero) {
         auto a = xmm1;
         auto b = xmm2;
         auto c = xmm3;
 
         uni_vmovdqu(a, xword[src]);          // load 4 floats
         uni_vmovdqu(b, a);                   // b = a
         uni_vmovdqu(c, a);                   // c = a
-        uni_vpcmpeqd(b, b, zero);               // if (a == 0) b = 1 else b = 0
-        uni_vpand(c, c, mask);                  // c = a & 01111111100000000000000000000000
-        uni_vpcmpeqd(c, c, zero);               // if (c == 0) c = 1 else c = 0
-        uni_vtestps(b, c);                    // if ((!b & c) == 0) CF = 1 else CF = 0
+        uni_vpand(b, b, mantissa_mask);      // b = a & 00000000011111111111111111111111
+        uni_vpcmpeqd(b, b, zero);            // if (a == 0) b = 1 else b = 0
```

Comment:
About comments. should be "if (b == 0) b = 1 else b = 0" ?

Reply:
Agree. Better to fix the comment.
Will do on master branch PR, just to avoid restarting the CI here.

Reply:
@luweizhou2016 Your comment is addressed here: https://github.com/openvinotoolkit/openvino/pull/19360/commits/178e3e29bcc6ba4372cfb8928e020419ec3bc0ac